ci/vic_install_utils

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ -z "$WORKDIR" ]; then
+    export WORKDIR=$HOME/workdir
+    mkdir -p $WORKDIR
+fi
+
+function install_mpich {
```

Comment:
I don't think so.  Eventually, `mpich` or `open-mpi` will be whitelisted and we can stop building it outselves.